doc/src/performance_tuning.rst

```diff
@@ -0,0 +1,151 @@
+.. highlight:: none
+
+******************
+Performance Tuning
+******************
+
+OpenMP Backend
+==============
+
+libxsmm
+-------
+
+Although PyFR will usually function without libxsmm it is nevertheless
```

Comment:
Maybe add a one-liner about what libsxmm does and add a link to the git repo

Reply:
Updated with description of libxsmm, link is already in the installation guide (along with the specifics about how it needs to be compiled)